frontend/stylesheets/main.less

```diff
@@ -110,6 +110,14 @@ body {
   }
 }
 
+.ant-alert-message {
+  margin-left: 5px;
+}
+.ant-alert-banner {
+  position: absolute;
+  z-index: 1000;
+  width: 100%;
+}
```

Comment:
These are antd classes, right? Please move these (and the rule in line 121f) to `antd_overwrites.less`.

Reply:
okay! when does something count as overwriting? 
there are multiple CSS rules for antd in main.less, e.g. in l. 72 and I am not sure I understand when to write antd rules where

Reply:
You are right, it's not very clear right now. I moved another rule, but the others rules mostly use antd class names in combination with other descriptors (e.g., `#main-container > section > .ant-layout-header`). This means that only specific occurrences of antd components are changed. In the overwrites module, on the other hand, the rules are mutating all occurrences of the mentioned antd components. So, this is can be used as a rule of thumb :)